packages/components/src/__actions__/Button/v3/_docs/Button.spec.stories.tsx

```diff
@@ -1,131 +1,126 @@
 import React from "react"
 import { Meta, StoryObj } from "@storybook/react"
 import { userEvent, waitFor, within, expect, fn } from "@storybook/test"
-import { VisuallyHidden } from "react-aria"
-import {
-  AddIcon,
-  TrashIcon,
-  ThumbsUpOffIcon,
-  ThumbsUpOnIcon,
-} from "~components/Icon"
-import { Tooltip, TooltipTrigger } from "~components/__overlays__/v3"
+import { VisuallyHidden } from "~components/VisuallyHidden"
+import { Icon } from "~components/__future__/Icon"
 import { Button } from "../index"
 
 const onPressEvent = fn()
 
 const meta = {
-  title: "Actions/Button/Button (v3)/Tests",
+  title: "Actions/Button/Button (v3)/Button (v3) tests",
   component: Button,
   args: {
     children: "Label",
     onPress: onPressEvent,
   },
+  parameters: {
+    chromatic: { disable: false },
+  },
```

Comment:
We should not default to sending to Chromatic for any stories that aren't stickersheets. Especially here, it seems they are more behavioural tests, so they should just be tested in CI instead.